### PR TITLE
show message when click on each chat room

### PR DESCRIPTION
**Pull request by yongsokheng**

### CODE REVIEW

File: app/assets/javascripts/chat.js [hunk 1]
```diff
@@ -8,4 +8,14 @@ $(document).on("turbolinks:load", function() {
     height: "86%",
     size: "6px"
   });
+
+  $(".chat-room").on("click", function() {
```
Comment (nguyenngocha) [inline app/assets/javascripts/chat.js]: single quote

File: app/controllers/messages_controller.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+class MessagesController < ApplicationController
+  def new
+    chat_room_id = params[:id]
+    chat_room_type = params[:type]
+
+    if chat_room_type == "User"
```
Comment (nguyenngocha) [inline app/controllers/messages_controller.rb]: settings

File: app/assets/javascripts/chat.js [hunk 3]
```diff
@@ -9,3 +9,15 @@ $(document).on("turbolinks:load", function() {
...
+$(document).on("turbolinks:load ajaxComplete", function() {
+  $(".chat-room").on("click", function() {
+    var chat_room_id = $(this).data("id");
+    var chat_room_type = $(this).data("type");
+
+    $(".chat-room").removeClass("active");
+    $(this).addClass("active");
+
+    $.get("/messages/new", {id: chat_room_id, type: chat_room_type});
+  });
```
Comment (NgengChhengkim) [inline app/assets/javascripts/chat.js]: dung ' '

**Comment by NgengChhengkim** [comment]
OK

**Pull request merged**